actions/user_actions.jsx

```diff
@@ -697,3 +697,33 @@ export async function sendPasswordResetEmail(email, success, error) {
         error({id: err.server_error_id, ...err});
     }
 }
+
+/**
+ * Send a getUserAnalytics request for a user (userId) and team id (teamId)
+ * Pass a string indicating which dataset to return (name)
+ *
+ * @param {string} userId - The target user for the requested analytics
+ * @param {string} teamId - The target team for the requested analytics
+ * @param {string} name - The name of the type of analytic data to return
+ *
+ * @returns {AsyncThunk<object>} that resolves with the requested analytics
+ */
+export async function getUserAnalytics(userId, teamId, name) {
```

Comment:
Is this function still being used anywhere, or is this an artifact of the rebase?

Reply:
I think its an ancient artifact.